README.md

```diff
@@ -1,8 +1,6 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-light-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
-  <img alt="Shows an illustrated light mode meshery logo in light color mode and a dark mode meshery logo dark color mode." src="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-tag-light-text-side.png" width="70%" align="center" style="margin-bottom:20px;">
-</picture></a><br /><br /></p>
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture><img src="./.github/assets/images/readme/meshery-logo-light-text-side.svg#gh-dark-mode-only" width="70%"/>
+<img src="./.github/assets/images/readme/meshery-logo-dark-text-side.svg#gh-light-mode-only"
```

Comment:
Please use ```<picture>``` tag something like

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/meshery/meshery/blob/master/.github/assets/images/Meshery-celebration-Dark.svg">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/meshery/meshery/blob/master/.github/assets/images/Meshery-celebration-Light.svg">
  <img alt="Meshery Logo" src="">
  </picture>
```

The ```#gh-dark-mode-only``` thing doesn't work outside of the GitHub, since GitHub readme's are used in NPM package page, for example.

Reply:
It is officially deprecated by GitHub https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

Reply:
Thank you @anshgoyalevil for your feedback and suggestions . I ll have a 👀. I was already thinking about finding a different method.